src/components/Cards/VotingTableCardRow/VotingTableCardRow.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import DecisionResult from "../../Shared/DecisionResult";
+import { MATTER_STATUS_DECISION } from "../../../constants/ProjectConstants";
+import { VOTE_DECISION } from "../../../constants/ProjectConstants";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+type VotingTableCardRowProps = {
+  /** the name of the matter that was voted on */
+  legislationName: string;
+  /** whether or not the row is an even or odd row */
+  isEven: boolean;
+  /** the persons vote */
+  voteDecision: MATTER_STATUS_DECISION;
+  /** the voting body decision */
+  councilDecision: VOTE_DECISION;
+  /** link to the detail page of the matter being voted on */
+  legislationLink: string;
+  /** words indicating the topic of the matter being voted on */
+  legislationTags: string[];
+  /** date of the matter being voted on */
+  meetingDate: string;
+  /** link to the detail page of the meeting */
+  meetingLink: string;
+  /** name of the voting body */
+  committeeName: string;
+};
+
+const VotingTableCardRow = ({
+  isEven,
+  legislationLink,
+  legislationName,
+  legislationTags,
+  voteDecision,
+  councilDecision,
+  meetingDate,
+  meetingLink,
+  committeeName,
+}: VotingTableCardRowProps) => {
+  const backgroundColor = isEven ? "rgba(148,148,148,0.2)" : "white";
+  const legislationTagsString =
+    legislationTags && legislationTags.length > 0 ? legislationTags.join(` • `) : "";
```

Comment:
Note to self: We should make this dot character be a constant somewhere instead of copy pasting it around to all the files.

Reply:
I made a StyleConstants file and ` • ` is the first (and thus far only) thing in it.  Replaced all instances of ` • ` in the project to TAG_CONNECTOR so we can change it to an emoji later.